code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -192,11 +201,15 @@ Please contact me on #coderbus IRC. ~Carnie x
 
 /mob/living/carbon/human/update_inv_wear_id()
 	remove_overlay(ID_LAYER)
+
 	if(wear_id)
 		if(client && hud_used && hud_used.hud_shown)
 			wear_id.screen_loc = ui_id
 			client.screen += wear_id
 
+		if(dna && istype(dna.species, /datum/species/pony))

```

Comment:
It's a race that would be used only once, for one day. I could make it less snowflake, and I know how to do this, but I'm not going to bother actually doing this. Unless /tg/station would want to keep ponies permanently, which clearly wouldn't happen.